production/README.md

```diff
@@ -8,7 +8,7 @@ Currently there are five ways to try out Loki, in order from easier to hardest:
 - [Build Loki from source](#build-and-run-from-source)
 - [Get inspired by our production setup](#get-inspired-by-our-production-setup)
 
-For the various ways to run `promtail`, the tailing agent, see our [Promtail documentation](../docs/clients/promtail/README.md).
+For the various ways to run `promtail`, the tailing agent, see our [Promtail documentation](https://grafana.com/docs/loki/latest/clients/promtail/installation/).
```

Comment:
Can we stick to the relative pathing instead?

Reply:
Thank you for your comment! I've replaced the link with the relative path to a file with installation instruction. However, I doubt its quality. Could you please double-check if this is what you meant?
Commit with update — b9c18a19f2a926fd397323ecf941ea4c57a73b0b